src/types.ts

```diff
@@ -100,13 +100,16 @@ export const JSONRPCResponseSchema = z
   .strict();
 
 /**
- * An incomplete set of error codes that may appear in JSON-RPC responses.
- */
+* @author : Sumitesh Naithani
+* @link : https://docs.trafficserver.apache.org/en/latest/developer-guide/jsonrpc/jsonrpc-node-errors.en.html#standard-errors
+* @description : An incomplete set of error codes that may appear in JSON-RPC responses.
+* @note : SDK-specific errors should use the server error range (-32000 to -32099), as per JSON-RPC 2.0 specification.
```

Comment:
We're not using a documentation generator that will make use of these kind of @ annotations right now. Can you please reformat as a normal documentation comment?